_includes/week.md

```diff
@@ -3,7 +3,7 @@
 ## {{ c.type }} call: {{ c.title }}
 
 {% if c.type != "Mentor-Mentee" and c.date %}
-<i class="fas fa-calendar-alt"></i> **Date**: {{ c.date }}{% if c.time %}, at [{{ c.time }} Universal Time](https://arewemeetingyet.com/UTC/{{ c.date | date: "%Y-%m-%d" }}/{{ c.time }}/
```

Comment:
Can you revert all your changes on the time? Thanks

Reply:
Oh wow. I only just noticed the time changes now. Will revert immediately, please...

Reply:
Done. 👍

Reply:
What do you mean?